metrics/network/remote_network/README.md

```diff
@@ -0,0 +1,42 @@
+# Clear Containers remote networking test
+
+- `remote networking iperf3` measures bandwidth using iperf3 using a remote setup. Host A will run 
+a container that will act as a server while host B will run a container that will act as a client 
+and the network bandwidth will be measured across the containers.
+
+## Prerequisite
+An automatic login from host A / user a to host B /user b is needed. It is necessary to setup authentication 
```

Comment:
Let's be consistent in capitalizing "A" and "B":
...host A (user A) to host B (user B) is needed. 
Also, let's change to use active voice:
s/It is necessary to setup/You must setup authentication keys to do an ssh login without a password.